tiledb/sm/query/query.h

```diff
@@ -780,7 +817,36 @@ class Query {
    */
   bool has_aggregates() {
     // We only need to check the default channel for now
-    return default_channel_aggregates_.empty();
+    return !default_channel_aggregates_.empty();
+  }
+
+  /**
+   * Returns the number of actual channels. "Actual channels" is the replacement
+   * for the current channel class.
+   *
+   * Responsibility for choosing channel identifiers is the responsibility of
+   * this class. At the present time the policy is very simple, since all
+   * queries only draw from a single array.
+   *   - Channel 0: All rows from the query. Always non-segmented, that is,
+   *       without any grouping.
+   *   - Channel 1: (optional) Simple aggregates, if any exist.
+   */
+  inline size_t number_of_actual_channels() {
```

Comment:
If we do the rename below we might be able to get rid of the word actual?

Reply:
Sure. I did it that way to avoid conflating a bunch of renames with the rest of the PR.